grid/example/src/cells/RowOptions.js

```diff
@@ -18,7 +20,17 @@ const RowOptions = memo((props) => {
         setOpen(false);
     };
 
-    return (
+    const showEditOverLay = () => {
+        console.log("showEditOverLay");

```

Comment:
Console logs should not be present in the production repo

Reply:
Completed